### ISSUE TITLE
UI: verified profile in NostrGit

### ISSUE THREAD

**Issue by ghost** [NostrGit/NostrGit#233]
Hello,

I would like to know what you all already thought about adding verified profile support to the NostrGit. Have you all thought about adding verified profile to NostrGit?

For example,
![image](https://user-images.githubusercontent.com/131187009/232861778-50684038-9e8d-44b4-b6ac-96fc24b51bb6.png)

*what do you all think of this idea?*

**Comment by cypherhoodlum** [comment]
Yes, [NIP-05](https://github.com/nostr-protocol/nips/blob/master/05.md) should of course be implemented for all profiles. Thanks for the Issue.
